Testing/CI/Azure/azure.cmake

```diff
@@ -1,11 +1,10 @@
-
 # set_from_env
 # ------------
 #
 # Sets a CMake variable from an environment variable. If the
```

Comment:
Looks like the original file was correct and the modified one has the grammar issues.

Reply:
Thanks for detecting some rebasing issues.